### PR TITLE
feat(presets): local bitbucket-server presets

### PR DESCRIPTION
**Pull request by viceice**
Allow using shared local presets for bitbucket-server

### CODE REVIEW

**Comment by viceice** [comment]
review? 🙃

File: lib/config/presets/bitbucket-server/index.ts [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  endpoint: string
+): Promise<Preset> {
+  const [projectKey, repositorySlug] = repo.split('/');
+  setBaseUrl(endpoint);
+  const url = `/rest/api/1.0/projects/${projectKey}/repos/${repositorySlug}/browse/${fileName}?limit=20000`;
+  let res: { body: FileData };
+  try {
+    res = await http.getJson(url);
+  } catch (err) {
+    console.error({ url, err });
```
Comment (rarkins) [inline lib/config/presets/bitbucket-server/index.ts:25]:
```suggestion
    logger.warn({ url, err }, 'Error fetching JSON file');
```

**Review by rarkins — CHANGES_REQUESTED** [review]

File: lib/config/presets/bitbucket-server/index.ts (hunk 1 above)
Comment (viceice) [inline lib/config/presets/bitbucket-server/index.ts:25]: ups, testing leftover 🙈 
```suggestion
```

**Comment by viceice** [comment]
fixed 🙃

**Review by rarkins — APPROVED** [review]

**Review by JamieMagee — APPROVED** [review]

**Pull request merged**